pityi.envf

```diff
@@ -0,0 +1,7 @@
+DB_HOST=0.0.0.0
+DB_PORT=3306
+DB_USERNAME=bulletin
```

Comment:
What is this file for? :D

Btw do you still need the `.env` file since dev environment is in docker now? :)